kafka/4.6/modules/ROOT/pages/index.adoc

```diff
@@ -28,6 +28,60 @@ An example is a newsroom that uses the Apache Kafka system to deliver the latest
 * Multiple applications that rely on the information provided by Apache Kafka +
 For example, a department store uses a website activity tracker to improve the online shopping experience. The data that is gathered is sent to multiple departments for various computations. Each department reviews the information that's received to stay informed about what the customer is looking for and then provides recommendations accordingly.
 
+=== Publish-to Use Cases
+
+With Kafka Connector, applications can publish binary messages to the Kafka server. Some examples of use cases include:
+
+* Users publishing binary data to the Kafka server and the underlying topic of their choice.
+* Integration specialists working with large volumes of data who want to set the batch size of the data to publish to Kafka.
+** To optimize for performance, companies can batch messages to send. Kafka stores the message until the batch size is reached and completes the send at that time.
+* Users focusing on optimal performance who want to configure the message size that they send.
+** Kafka is not intended to process and handle large messages. Messages are typically in the 1 MB range and users set the size to avoid performance degradation. In the case of an error, the message is rejected.
+* Users ensuring successful data transmission and want to block further commits on a configurable level of acknowledgment for the messages published to the Kafka server.
+** Acknowledgments enable users to ensure that the message is sent prior to publishing more messages. This is specifically useful in time-ordered or event-ordered messages, such as logs or patient drug administration orders. Ensuring that messages are sent in order helps the downstream application process the messages in the right order.
+* Users configuring how data is serialized during a publish.
+* Users securing messages sent during Mutual TLS.
+
+The following lists the acceptance criteria for publishing:
+
+* Publish a message to a Kafka topic.
+* Configure the batch size and size of the message.
+* Obtain an acknowledgment once publishing is complete at the configured level.
+* Send the message securely via Mutual TLS.
+* Configure the serialization type.
+
+=== Subscribe-from Use Cases
+
+With Kafka Connector, applications can obtain messages from Kafka for further consumption downstream. Some examples of use cases include:
+
+Reading from Kafka:
+
+* Users subscribing to the Kafka server and the underlying group of their choice to read and consume messages for downstream processing.
+* Users subscribing to a specific Kafka partition to process messages.
+** Users with a high availability microservice don't want to rely on Kafka's rebalancing in the event of a failure.
+** If users have a local state associated with their partition, they want to process only that partition.
+* Users configure the value of their offset to a set value or the beginning or the end of the topic assigned to them.
```

Comment:
To match the others: Users configuring the value of their offsets to a set value or to the beginning or end of the topic assigned to them.